presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotConfig.java

```diff
@@ -468,4 +470,18 @@ public PinotConfig setUsePinotSqlForBrokerQueries(boolean usePinotSqlForBrokerQu
         this.usePinotSqlForBrokerQueries = usePinotSqlForBrokerQueries;
         return this;
     }
+
+    public int getLimitToPushdownOrderByForBrokerQueries()
+    {
+        return this.limitToPushdownOrderByForBrokerQueries;
```

Comment:
nit: limitThreshholdForTopNBrokerQueries ? 

To be more explicit that this covers order-by-limit and not unlimited order by. 

(And corresponding changes to the other DEFAULT_ and derived strings from this config name).

Reply:
done.